snafu/sysbench_wrapper/trigger_sysbench.py

```diff
@@ -40,27 +47,33 @@ def _run_sysbench(self):
         specific sysbench versions/future changes.
         """
 
-        #  open config file
-        config_file = open(self.sysbench_file)
+        #  Use args from command line or a config file
+        if self.sysbench_args:
+            config_options = self.sysbench_args.split()
+        elif self.sysbench_file:
+            config_options = open(self.sysbench_file, encoding="utf8")
+        else:
+            logger.error("Arugments were not specified by file or command line")
+            sys.exit(1)
 
         cmd = "sysbench"
         #  for each option add it to the command line
-        for option in config_file:
+        for option in config_options:
```

Comment:
If the user passes a file, all this will be is a pointer to the file, we need to `read` and `split` I would presume?

Reply:
Just tested and it works. This was the code prior to me adding the arguments and that worked.